sdk/storage/azure-storage-queue/samples/queue_samples_message.py

```diff
@@ -199,8 +204,10 @@ def receive_one_message_from_queue(self):
             # We should see message 3 if we peek
             message3 = queue.peek_messages()[0]
 
-            print(message1.content)
-            print(message2.content)
+            if message1 is not None and hasattr(message1, 'content'):
```

Comment:
Hmm don't love these checks. I suppose `is not None` is required since `receive_message`'s return is optional. Maybe we should None check and throw if either is None above? As for `hasattr`, that shouldn't be needed as a `QueueMessage` has an attribute `content`. Maybe the possibility of None was throwing the error but once we confirm its not None, MyPy should be smart enough to figure it out.